docs/web_api.md

```diff
@@ -1607,6 +1607,59 @@ Currently we support Slic3r derivatives and Cura with Cura-Octoprint.
   }
   ```
 
+## History APIs
+The APIs below are avilable when the `[history]` plugin has been configured.
+
+### Get job list
+- HTTP command:\
+  `GET /printer/histor/list?limit=50&start=50&since=1&before=5`
+
+- Websocket command:\
+  `{"jsonrpc":"2.0","method":"printer.history.list","id":"1","params":{}}`
+
+All arguments are optional. Arguments are as follows:
+`before` All jobs before this UNIX timestamp
+`limit` Number of prints to return
+`since` All jobs after this UNIX timestamp
+`start` Record number to start from (i.e. 10 would start at the 10th print)
```

Comment:
I believe that this endpoint also accepts an "id" argument that returns a single job.